docs/src/hugo/content/getting-started/first-use.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: "single"
+title: First Use
+preamble: >
+  Launching your first system using Nelson is remarkably easy! This guide walks you through how to get-going with Nelson and covers the high-level components needed to get an end to end solution.
+contents:
+- Assumptions
+- Manifest
+- Enabling Nelson
+- Triggering a Deployment
+- Working with Stacks
+- Further Reading
+menu:
+  main:
+    parent: gs
+    identifier: gs-first-use
+    url: /getting-started/first-use.html
+    weight: 2
+---
+
+## Assumptions
+
+Whilst Nelson is typically non-perscriptive, for the unfamiliar user and for the remainder of this article, the following assumptions will be made to avoid confussion:
+
+* You can launch any container you like using Nelson; there is no requirement from the system perspective that the container behaves in a particular way other than having a [typical Nelson entrypoint](https://docs.docker.com/engine/reference/builder/#entrypoint).
```

Comment:
"typical Nelson entrypoint" may be confusing, we should just say that we expect a Docker entrypoint or command that does the right thing upon being run. May also be worth noting Nelson does not allow service owners to specify additional arguments or otherwise override the command during deployment, and therefore the command should transparently work from environment to environment. This typically means using one of the Nelson environment variables like `${NELSON_ENV}` to say, render config/secrets.